lib/component.js

```diff
@@ -314,14 +323,27 @@ class Component extends WebComponent {
     this.initialized = false;
   }
 
+  /**
+   * Attributes schema that defines the component's attributes and their types
+   * Panel auto parses attribute changes into this.attrs object and $attrs template helper
+   *
+   * @type {Object.<string, AttrSchema>}
```

Comment:
can't document the schema format here?

Reply:
I did it, but it didn't show up in jsdoc :/ So I kept it simple. I'll put it back.